tests/routes/socialAuth.test.js

```diff
@@ -0,0 +1,270 @@
+/* eslint-disable no-unused-expressions */
+import chai, { should } from 'chai';
+import chaiHttp from 'chai-http';
+import { config } from 'dotenv';
+import axios from 'axios';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import { OAuth2Client } from 'google-auth-library';
+
+import db from '../../src/models';
+import app from '../../src';
+import { performCallback, createOrFindUser } from '../../src/services/auth';
+import authController from '../../src/controllers/auth';
+
+chai.use(chaiHttp);
+chai.use(sinonChai);
+should();
+config();
+
+const { twitterLogin } = authController;
+
+before(async () => {
+  try {
+    await db.sync({ force: true });
+  } catch (error) {
+    return error;
+  }
+});
+
+describe('Social Login Route', () => {
+  it('should throw error social provider is invalid', async () => {
+    const response = await chai.request(app).post('/api/v1/auth/something');
+
+    response.should.have.status(400);
+    response.body.status.should.eql('error');
+  });
+
+  it('should throw error if access token is invalid', async () => {
+    const invalidToken = { accessToken: 'invalid access token' };
+    const response = await chai
+      .request(app)
+      .post('/api/v1/auth/google')
+      .send(invalidToken);
+
+    response.should.have.status(500);
+    response.body.status.should.eql('error');
+  });
+
+  describe('facebook Authentication method', () => {
+    let facebook;
+    let sampleToken;
+
+    beforeEach(() => {
```

Comment:
Is the indentaion 1 tab/2 spaces? Looks like 4. Please recheck

Reply:
I think it's 2 though

Reply:
It's 2 spaces I used